.pre-commit-config.yaml

```diff
@@ -3,3 +3,5 @@ repos:
   rev: 18.9b0
   hooks:
   - id: black
+    # Doesn't work:
+    # args: [--target-version py35]
```

Comment:
pre-commit/black formats the files as Python 3.6 but this is incompatible with Python 3.5 and the above config doesn't work.
See the `# fmt: off` comments in `repo2docker/docker.py` for the alternative workaround.

Reply:
Hopefully https://github.com/jupyter/repo2docker/pull/849 will fix this